exercise5.js

```diff
@@ -0,0 +1,17 @@
+
+function calculateFactorial(n) {
+    // if (n === 0 || n === 1) {
+    //   return 1;
+    // } else {
+    //   var factorial = 1;
+    //   for (var i = 2; i <= n; i++) {
+    //     factorial *= i;
+    //   }
+    //   return factorial;
+    // }
+  
+    return n === 0 || n === 1 ? 1 : Array.from({length: n}, (v, i) => i + 1).reduce((a, b) => a * b);
+    
```

Comment:
I like the conciseness! One recommendations I would make is similar to #1 which is to rename the variable names to reflect their meaning. variables like v, i, a and b.